libs/ipfs-http-client/Makefile

```diff
@@ -0,0 +1,74 @@
+# 
+# This is free software, licensed under the GNU General Public License v2.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ipfs-http-client
+PKG_RELEASE:=1
+
+PKG_MAINTAINER:=Leonid Esman <leonid.esman@gmail.com>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE.MIT
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/vasild/cpp-ipfs-http-client.git
+#PKG_SOURCE_DATE:=2019-11-05
```

Comment:
This should be uncommented, no?

Reply:
I am not sure, this only affects the package name, but not the program functionality.

Reply:
Many packages use PKG_SOURCE_DATE. Let it be.